rholang/src/test/scala/coop/rchain/rholang/InterpreterSpec.scala

```diff
@@ -27,27 +27,26 @@ class InterpreterSpec extends FlatSpec with Matchers {
 
     val sendRho = "@{0}!(0)"
 
-    val (initStorage, beforeError, afterError, afterSend, finalContent) =
-      mkRuntime[Task](tmpPrefix, mapSize)
-        .use { runtime =>
-          for {
-            initStorage  <- storageContents(runtime)
-            _            <- success(runtime, sendRho)
-            beforeError  <- storageContents(runtime)
-            _            <- failure(runtime, "@1!(1) | @2!(3.noSuchMethod())")
-            afterError   <- storageContents(runtime)
-            _            <- success(runtime, "new stdout(`rho:io:stdout`) in { stdout!(42) }")
-            afterSend    <- storageContents(runtime)
-            _            <- success(runtime, "for (_ <- @0) { Nil }")
-            finalContent <- storageContents(runtime)
-          } yield (initStorage, beforeError, afterError, afterSend, finalContent)
-        }
-        .runSyncUnsafe(maxDuration)
-
-    assert(beforeError.contains(sendRho))
-    assert(afterError == beforeError)
-    assert(afterSend == beforeError)
-    assert(finalContent == initStorage)
+    mkRuntime[Task](tmpPrefix, mapSize)
+      .use { runtime =>
+        for {
+          initStorage           <- storageContents(runtime)
+          _                     <- success(runtime, sendRho)
+          beforeError           <- storageContents(runtime)
+          _                     = assert(beforeError.contains(sendRho))
+          beforeErrorCheckpoint <- runtime.space.createCheckpoint()
```

Comment:
These checkpoints are called in `Interpreter.evaluate`. Why are you calling them explicitly?

Reply:
They are called before the evaluation. I'm asserting on checkpoints created after the evaluation